src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ReviewSubmittedEvidence.cshtml

```diff
@@ -40,7 +40,7 @@
                             {
                                 <tr class="govuk-table__row">
                                     <td class="govuk-table__cell">@note.ReferenceDisplay</td>
-                                    <td class="govuk-table__cell">@(note.SubmittedDate.HasValue ? note.SubmittedDate.Value.ToShortDateString() : string.Empty)</td>
+                                    <td class="govuk-table__cell">@(note.SubmittedDate.HasValue ? note.SubmittedDate.Value.ToShortDateString() : "-")</td>
```

Comment:
Would be tempted to move this display logic to the view model. Then we can test it.